backend/spec/factories/groups.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :group do
+    name { 'MyString' }
+    user_id { 1 }
```

Comment:
Let's create user here directly ([docs](https://www.rubydoc.info/gems/factory_bot/file/GETTING_STARTED.md#associations))

Reply:
This factory is not working. Please, make sure it creates group correctly
```ruby
FactoryBot.create(:group)
```